### PR TITLE
Merge Freeze into Master (3003.2)

### PR DESCRIPTION
**Pull request by Ch3LL**

### CODE REVIEW

**Review by krionbsd — DISMISSED** [review]

File: tests/pytests/unit/loader/test_context.py [hunk 1]
```diff
@@ -1,47 +0,0 @@
-"""
-Tests for salt.loader.context
-"""
-import copy
-
-import salt.loader.context
-import salt.loader.lazy
-
-
-def test_named_loader_context():
```
Comment (s0undt3ch) [inline tests/pytests/unit/loader/test_context.py:10]: I believe this test file should stay?

**Review by s0undt3ch — CHANGES_REQUESTED** [review]

**Comment by Ch3LL** [comment]
re-run full all

**Review by s0undt3ch — DISMISSED** [review]

File: salt/modules/saltutil.py [hunk 2]
```diff
@@ -89,6 +89,8 @@ def _get_top_file_envs():
                 raise CommandExecutionError(
                     "Unable to render top file(s): {}".format(exc)
                 )
+            finally:
+                st_.client.destroy()
```
Comment (dwoz) [inline salt/modules/saltutil.py:93]: This is no longer needed because HighState is being used as a context manager.

**Comment by Ch3LL** [comment]
Closing as these changes where included in this merge: https://github.com/saltstack/salt/pull/60924

**Pull request closed without merging**